checkers/checkers_test.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
+func init() {
+	if err := InitEmbeddedRules(); err != nil {
```

Comment:
Is there a sense to keep this func exported after this PR ?

I'm not proposing to drop it, more like deprecated and remove in the future. WDYT?

Reply:
I believe golangci-lint may still want to avoid registering the embedded rules if gocritic is disabled.